content/en/install.md

```diff
@@ -50,43 +55,87 @@ Other options include:
   and the IEP interactive development environment; Supports Linux,
   Windows, and Mac.
 
+If you already have Python installed, you can
+[install SciPy with `pip`](#pip-install).
+For more advanced users who will need to install or upgrade regularly,
+you may prefer to use a dedicated package manager like [`uv`]
+(see [Installing with `uv`](#uv-install)),
+[Conda], or [`pixi`] (see [Installing from conda-forge](#conda-forge-install)).
+
+[`uv`]: https://docs.astral.sh/uv/
+[Conda]: https://docs.conda.io/projects/conda/en/latest/index.html
+[`pixi`]: https://pixi.sh/latest/
+
+<a name="pypi-install"></a>
+
+## Installing from PyPI
```

Comment:
Is the _source_ of the installation the major thing to split on here? Or should it be more like:

- Installing into an environment
   - Using pip, uv, or conda
- Adding to your project
    - Using uv or pixi
- Installing system-wide
    - ...

Reply:
That sounds pretty good to me! Although it is more of a departure from the existing text